lib/core/utils/preload-media.js

```diff
@@ -14,6 +14,27 @@ function preloadMedia({ treeRoot = axe._tree[0] }) {
     treeRoot,
     'video, audio',
     ({ actualNode }) => {
+      /**
+       * Ignore media that won't load no matter how long we wait
+       * @see https://github.com/dequelabs/axe-core/issues/4665
+       */
+      if (
+        actualNode.preload === 'none' &&
+        actualNode.networkState !== actualNode.NETWORK_LOADING
```

Comment:
should we ignore the `preload="metadata"` too? it seems like it doesn't put the media in a ready to play state either

Reply:
We don't need to wait for the media to be playable, we just need the metadata so we can identify the duration (that's why `isMediaElementReady` waits for any non-zero ready state, rather than specifically waiting for a contentful ready state)